src/Query/FromClause.php

```diff
@@ -32,13 +32,12 @@ public function aliasRecentExpression(string $name): void {
   /**
    * The FROM clause of the query gets processed first, retrieving data from tables, executing subqueries, and handling joins
    * This is also where we build up the $columns list which is commonly used throughout the entire library to map column references to indexes in this dataset
+   * @reviewer, we don't build up the $columns, since the variable is unused...
```

Comment:
ah yeah, this is from an older version of this before schema was present I think!